tests/integration/package/package_integ_base.py

```diff
@@ -14,7 +14,22 @@ class PackageIntegBase(TestCase):
     @classmethod
     def setUpClass(cls):
         cls.region_name = os.environ.get("AWS_DEFAULT_REGION")
```

Comment:
This should be below doc strings.

Reply:
Actually doc string is only for the line where we are reading `AWS_S3` and not for the whole method. That is why I didn't move this below docstring. Thoughts?